src/pdc-api.ts

```diff
@@ -62,25 +72,28 @@ interface Proposal {
 }
 
 const useProposal = (proposalId: string) => (
-  usePdcApi<Proposal>(`/proposals/${proposalId}?includeFieldsAndValues=true`)
+  usePdcApi<Proposal>(
+    `/proposals/${proposalId}`,
+    new URLSearchParams({
+      includeFieldsAndValues: 'true',
+    }),
```

Comment:
Just to be explicit, my suggestion above would mean this could just become:

```suggestion
    {
      includeFieldsAndValues: 'true',
    },
```